src/processing/dom/index.js

```diff
@@ -152,6 +152,8 @@ export default class DomProcessor {
         var replacer = (resourceUrl, resourceType, charsetAttrValue) => urlReplacer(resourceUrl, resourceType, charsetAttrValue, baseUrl);
         var $all     = $('*');
 
+        var ELEMENT_IS_PROCESSED = 'hammerhead|element-is-processed';
```

Comment:
This place will be fixed in separate issue - https://github.com/superroma/testcafe-hammerhead/issues/194